foundry.toml

```diff
@@ -20,6 +20,9 @@ remappings = [
     'forge-std/=lib/forge-std/src/'
 ]
 
+[profile.ci]
```

Comment:
how does this environment get triggered? i think github actions passes in `CI=true` automatically- is that how?

Reply:
I believe it is triggered by the Github CI action using:
```
env:
  FOUNDRY_PROFILE: ci
```